src/displayBackend/wayland/Display.cpp

```diff
@@ -104,21 +104,25 @@ void Display::stop()
 	}
 }
 
-bool Display::isZeroCopySupported() const
+void Display::flush()
 {
 	lock_guard<mutex> lock(mMutex);
 
-#ifdef WITH_DRM
-	if (mWaylandDrm && mWaylandDrm->isZeroCopySupported())
-	{
-		return true;
-	}
-	if (mWaylandKms && mWaylandKms->isZeroCopySupported())
+	DLOG(mLog, DEBUG) << "Flush";
+
+	wl_display_roundtrip(mWlDisplay);
+#if 0
```

Comment:
either delete if 0 or define if/else with meaningful variable name

Reply:
It was rondtrip test. Need to be returned back to flush.